docs/rules/no-typed-store.md

```diff
@@ -1 +1,36 @@
 # no-typed-store
+
+> This rule disallows providing typings to the generic Store type.
+
+## Rule Details
+
+Typing the global `Store` is redundant, and selectors are type safe, so mentioning the state interface while injecting the store is unnecessary.
```

Comment:
```suggestion
Typing the global `Store` is redundant because selectors are type safe, so adding the generic state interface while injecting the store is unnecessary.
```